MANIFEST.in

```diff
@@ -0,0 +1,7 @@
+# This file specifies which non-py files should be included in Python package
+# that is generated by running 'python step.py sdist in build.py.
```

Comment:
This is a little unclear. Let's write:

"This file specifies which non-py files need to be included in the Oppia Python package that is generated in order for Apache Beam jobs to be able to run in production. Note that this package is generated by running `python step.py sdist` in build.py."

Or something like that, that gives more context.

Also why do we call it step.py and sdist? These names are not very clear and don't seem to be related to building a pip package. [EDIT: should step.py be setup.py?]

Finally, any thoughts about how to disambiguate this from manifest.json? I think it's likely that devs will be confused due to there being multiple manifests.

Reply:
It should be setup.py, these are commands that are required by Python. Not sure how to disambiguate, MANIFEST.in is required and cannot be renamed.

Reply:
We have control over manifest.json. Can we change its name?

Reply:
(Also adding @U8NWXD since this will affect https://github.com/U8NWXD/oppia-wiki/pull/35/files.)

Reply:
Yeah, we can I would do it in a different PR though.

Reply:
OK, please file an issue and let's track this as part of Py3 cleanup.

Reply:
Done #13965.